lib/libc/common/Kconfig

```diff
@@ -53,7 +53,6 @@ config COMMON_LIBC_MALLOC_ARENA_SIZE
 config COMMON_LIBC_CALLOC
 	bool "Common C library calloc"
 	depends on COMMON_LIBC_MALLOC
-	default n if MINIMAL_LIBC && !MINIMAL_LIBC_CALLOC
```

Comment:
You can just remove `&& !MINIMAL_LIBC_CALLOC` here. The goal is to disable calloc when using the minimal C library by default.

Reply:
updated